src/plugins/gui/scenes/EnterFormScene.tsx

```diff
@@ -0,0 +1,371 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
This scene is too overloaded. Best to break up into an AddressScene and SepaScene. This would greatly simplify the component